src/ukify/ukify.py

```diff
@@ -1277,8 +1277,7 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
         '--signtool',
         choices = ('sbsign', 'pesign'),
         dest = 'signtool',
-        default = 'sbsign',
-        help = 'whether to use sbsign or pesign. Default is sbsign.',
+        help = 'whether to use sbsign or pesign. It can also be inferred by the other parameters given: when using --secureboot-{private-key/certificate}, use sbsign, otherwise use pesign',
```

Comment:
This should be reworded slightly - "It will also be inferred... sbsign will be used, otherwise pesign will be used", otherwise it reads as if the user had to do something to make this automated selection happen